app.js

```diff
@@ -1,8 +1,45 @@
 // import functions
+import { people } from './data.js';
+
 
 // grab DOM elements
+const selectElement = document.querySelector('.select');
+
+const divContainer = document.querySelector('.contain');
+const name = document.getElementById('.name');
```

Comment:
this line has a small syntax error that is breaking your app. you can use either `document.getElementById('name');` to get it by id, 
or `document.querySelector('.name');` to get it by class, 
or `document.querySelector('#name');` to get it by id.